index.html

```diff
@@ -0,0 +1,359 @@
+                  beguiled and demoralized by the charms of pleasure of the
+                  moment, so blinded by desire, that.
+                </p>
+              </div>
+              <div class="col-12 WhyChooseUs_button">
+                <a href="#" class="button">Explore Places </a>
+              </div>
+            </div>
+          </div>
+          <div class="col-6 col-lg-6 col-sm-12 morph">
+            <div class="row">
+              <div class="col-lg-12 col-sm-6 first">
+                <img class="first" src="img/Image2.png" alt="house" />
+              </div>
+              <div class="col-lg-12 col-sm-6 second">
+                <img class="second" src="img/Image1.png" alt="mountains" />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="SearchPlaces">
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-search"></span></div>
+              <div class="col-12"><p>Search Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-check-circle"></span></div>
+              <div class="col-12"><p>Select Places</p></div>
+            </div>
+          </div>
+          <div class="col-4">
+            <div class="row">
+              <div class="col-12"><span class="icon-thumbs-up"></span></div>
+              <div class="col-12"><p>Book & Enjoy</p></div>
+            </div>
+          </div>
+          <object
+            class="dotted_line"
+            data="img/Line.svg"
+            type="image/svg+xml"
+          ></object>
+        </div>
+      </div>
+    </section>
+    <section class="firstSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-7 col-xl-6 col-sm-12 firstSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="firstSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 firstSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="secondSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-6"></div>
+          <div class="col-lg-7 col-xl-6 col-sm-12 secondSpain_block">
```

Comment:
Чтоб не делать пустую колонку, лучше использовать класс **offset**
```suggestion
          <div class="col-lg-7 col-xl-6 col-sm-12 offset-lg-5 offset-xl-6  secondSpain_block">
```